keys.ts

```diff
@@ -1 +1,2 @@
-export const LIMIT_DISCLOSURE_SIGNATURES = ["RsaSignature2018", "Ed25519Signature2020"];
\ No newline at end of file
+export const LIMIT_DISCLOSURE_SIGNATURES_SUITES =
```

Comment:
This is not going to work for a library, since the same lib can be used in multiple contexts of an application.

This needs to be a property without possibility for duplicates and with methods.

"bbs+" is not a signature suite to begin with. Please look at respective specs!

Reply:
https://w3c-ccg.github.io/ld-cryptosuite-registry/

Reply:
https://w3c-ccg.github.io/ldp-bbs2020/#proof-type

Reply:
Fixed!